packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -934,15 +934,19 @@ test.serial('writeGranulesFromMessage() writes a granule and marks as failed if
   });
 
   const dynamoGranule = await granuleModel.get({ granuleId });
+  const dynamoGranuleError = JSON.parse(dynamoGranule.error.errors);
   t.is(dynamoGranule.status, 'failed');
-  t.deepEqual(dynamoGranule.error.Error, 'Failed writing files to PostgreSQL.');
+  t.deepEqual(dynamoGranuleError.map((error) => error.Error), ['Failed writing files to PostgreSQL.']);
+  t.true(dynamoGranuleError[0].Cause.includes('AggregateError'));
```

Comment:
I'm not necessarily against it, but what is the benefit of this assertion?

Reply:
The benefit is to test against the entire error object we expect in the array https://github.com/nasa/cumulus/blob/jt/2733/packages/api/lib/writeRecords/write-granules.js#L271-L274